WireAPI/Sources/WireAPI/Models/FeatureConfig/ConferenceCallingFeatureConfig.swift

```diff
@@ -26,4 +26,8 @@ public struct ConferenceCallingFeatureConfig: Codable, Equatable {
 
     public let status: FeatureConfigStatus
 
+    /// If set to `true`, clients will use SFTs for 1:1 calls
+
+    public let useSFTForOneToOneCalls: Bool?
```

Comment:
We could make this non-optional and default to `false`.

Reply:
done thx